code/src/components/MessageList.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react'
+
+import moment from 'moment'
+
+export const MessageList = () => {
+  const MESSAGES_URL = 'https://wk11livesession.herokuapp.com/messages'
```

Comment:
You are fetching the messages from the wrong URL 😅. This URL is the one we used during the lessons, but the one for the project should be:
https://happy-thoughts-technigo.herokuapp.com/thoughts 💕